src/panel/pages/explorer/components/ListItem.tsx

```diff
@@ -163,22 +162,6 @@ const ChildrenName = styled.span`
   font-size: 13px;
 `;
 
-const Arrow = styled(ArrowIcon)`
```

Comment:
Are you cool with us keeping styles/elements like this colocated? 

If we're reusing the style/icon I'm down for pulling it out into a shared folder but for one-time use cases, it's much easier to colocate and then break out when reuse occurs.

Reply:
So I reused this on on the Schema view, would you prefer I don't put this stuff in the shared folder just for that?